src/test/java/com/box/sdk/BoxFolderTest.java

```diff
@@ -1360,6 +1360,34 @@ public void testUploadFileWithDescriptionSucceeds() throws IOException {
         Assert.assertEquals(fileDescription, file.getDescription());
     }
 
+    @Test
+    @Category(UnitTest.class)
+    public void testGetFolderItemsWithSort() throws IOException {
+        String result = "";
+        final String folderID = "12345";
+        final String sortField = "name";
+        final String folderItemsURL = "/folders/" + folderID + "/items/";
+
+        result = TestConfig.getFixture("BoxFolder/GetFolderItemsWithSort200");
+
+        WIRE_MOCK_CLASS_RULE.stubFor(WireMock.get(WireMock.urlPathEqualTo(folderItemsURL))
+                .withQueryParam("sort", WireMock.equalTo("name"))
+                .withQueryParam("direction", WireMock.equalTo(BoxFolder.SortDirection.ASC.toString()))
+                .withQueryParam("limit", WireMock.equalTo("1000"))
+                .withQueryParam("offset", WireMock.equalTo("0"))
+                .willReturn(WireMock.aResponse()
+                       .withHeader("Content-Type", "application/json-patch+json")
```

Comment:
This isn't really what the API returns, is it?  Should probably be `"application/json"`